### PR TITLE
Add helm chart

### PR DESCRIPTION
**Pull request by dkarnutsch**
In order to publish it on ArtifactHub it needs to exist in the repository

### CODE REVIEW

File: chart/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          ports:
+            - name: http
+              containerPort: 3000
+              protocol: TCP
+          env:
+            - name: "ORIGIN_URL"
+              value: "{{ .Values.swrCacheProxy.originUrl }}}"
+            - name: "CACHE_DIR"
+              value: "/mnt/cache"
+          livenessProbe:
```
Comment (Schreglmann) [inline chart/templates/deployment.yaml:45]: Is it smart for liveness Probe and readiness Probe to use the same endpoint?

File: chart/templates/hpa.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
```
Comment (Schreglmann) [inline chart/templates/hpa.yaml]: v2beta1 is depricated

File: chart/templates/serviceaccount.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "foo.serviceAccountName" . }}
```
Comment (Schreglmann) [inline chart/templates/serviceaccount.yaml]: Placeholder?

File: chart/values.yaml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
```
Comment (Schreglmann) [inline chart/values.yaml]: Default Value should be a realistic value

File: chart/values.yaml [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+ingress:
+  enabled: false
+  className: ""
+  annotations:
+    {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
```
Comment (Schreglmann) [inline chart/values.yaml:51]: should this be chart-example?

**Review by Schreglmann — CHANGES_REQUESTED** [review]

File: chart/templates/deployment.yaml (hunk 1 above)
Comment (dkarnutsch) [reply chart/templates/deployment.yaml:45]: No, but it was an intentional decision to do it anyways: https://github.com/vivid-planet/swr-cache-proxy/pull/19

File: chart/templates/hpa.yaml (hunk 2 above)
Comment (dkarnutsch) [reply chart/templates/hpa.yaml]: changed to v2 (coming from helm init)

File: chart/templates/serviceaccount.yaml (hunk 3 above)
Comment (dkarnutsch) [reply chart/templates/serviceaccount.yaml]: changed

File: chart/values.yaml (hunk 4 above)
Comment (dkarnutsch) [reply chart/values.yaml]: changed to 10 (value coming from helm init)

File: chart/values.yaml (hunk 5 above)
Comment (dkarnutsch) [reply chart/values.yaml:51]: value is coming from helm init – I would leave it for now

**Review by Schreglmann — APPROVED** [review]

**Pull request merged**